recurser/string-is
# add a use-as-input button

![Kapture 2022-01-21 at 14 52 16](https://user-images.githubusercontent.com/96322/150473592-ad1d29d0-64c3-4db1-936e-c087a7c1effd.gif)



## How to test the PR

Paste or type text in to convert, click the `Use as input` button, and ensure that the output is moved to the input text field.
